doc/7/essentials/events/index.md

```diff
@@ -116,6 +116,16 @@ Triggered whenever Kuzzle responds with an error
 
 `@param {object} request - Request that caused the error`
 
+## callbackError
+
+Triggered whenever a callback on an event throw an error
```

Comment:
This event should be emitted only when the notification handler returns a rejected promise